odh/base/monitoring/overrides/grafana-operator/overlays/dashboards/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: opf-monitoring
```

Comment:
I don't think we need this namespace field here.

Reply:
do we need a kustomization if we are doing overlays?

Reply:
fixed